DisplayI2C1.cs

```diff
@@ -0,0 +1,274 @@
+
+			/* Init 4-bit mode + 2 line */
+			PulseEnable(Convert.ToByte((1 << this._D5)));
+			PulseEnable(Convert.ToByte((1 << this._D7)));
+
+			/* Turn on display, cursor */
+			PulseEnable(0);
+			PulseEnable(Convert.ToByte((1 << this._D7) | (Convert.ToByte(turnOnDisplay) << this._D6) | (Convert.ToByte(turnOnCursor) << this._D5) | (Convert.ToByte(blinkCursor) << this._D4)));
+
+			ClearScreen();
+
+			PulseEnable(0);
+			PulseEnable(Convert.ToByte((1 << this._D6) | (Convert.ToByte(cursorDirection) << this._D5) | (Convert.ToByte(textShift) << this._D4)));
+		}
+
+
+		/**
+        * Turn the backlight ON.
+        **/
+		public void TurnOnBacklight()
+		{
+			this._backLight = 0x01;
+			this.SendCommand(0x00);
+		}
+
+
+		/**
+        * Turn the backlight OFF.
+        **/
+		public void TurnOffBacklight()
+		{
+			this._backLight = 0x00;
+			this.SendCommand(0x00);
+		}
+
+
+		/**
+        * Can print string onto display
+        **/
+		public void Print(string text)
+		{
+			for (int i = 0; i < text.Length; i++)
+			{
+				this.Print(text[i]);
+			}
+		}
+
+
+		/**
+        * Print single character onto display
+        **/
+		public void Print(char letter)
+		{
+			try
+			{
+				this.Write(Convert.ToByte(letter), 1);
+			}
+			catch (Exception e)
+			{
+
+			}
+		}
+
+
+		/**
+        * skip to second line
+        **/
+		public void GotoSecondLine()
+		{
+			this.SendCommand(0xc0);
+		}
+
+
+		/**
+        * goto X and Y 
+        **/
+		public void GotoXY(byte x, byte y)
+		{
+			this.SendCommand(Convert.ToByte(x | _LineAddress[y] | (1 << LCD_WRITE)));
+		}
+
+
+		/**
+        * Send data to display
+        **/
+		public void SendData(byte data)
+		{
+			this.Write(data, 1);
+		}
+
+
+		/**
+        * Send command to display
+        **/
+		public void SendCommand(byte data)
+		{
+			this.Write(data, 0);
+		}
+
+
+		/**
+        * Clear display and set cursor at start of the first line
+        **/
+		public void ClearScreen()
+		{
+			PulseEnable(0);
+			PulseEnable(Convert.ToByte((1 << this._D4)));
+			Task.Delay(5).Wait();
+		}
+
+
+		/**
+        * Send pure data to display
+        **/
+		public void Write(byte data, byte Rs)
+		{
+			PulseEnable(Convert.ToByte((data & 0xf0) | (Rs << this._Rs)));
+			PulseEnable(Convert.ToByte((data & 0x0f) << 4 | (Rs << this._Rs)));
+			//Task.Delay(5).Wait(); //In case of problem with displaying wrong characters uncomment this part
+		}
+
+
+		/**
+        * Create falling edge of "enable" pin to write data/inctruction to display
+        */
+		private void PulseEnable(byte data)
+		{
+			this._i2cPortExpander.Write(new byte[] { Convert.ToByte(data | (1 << this._En) | (this._backLight << this._Bl)) }); // Enable bit HIGH
+			this._i2cPortExpander.Write(new byte[] { Convert.ToByte(data | (this._backLight << this._Bl)) }); // Enable bit LOW
+																											  //Task.Delay(2).Wait(); //In case of problem with displaying wrong characters uncomment this part
```

Comment:
This comment seems to be wrongly aligned

Reply:
I'll update all the comments with the next PR. This one was making sure you agreed with my changes, we can tidy up the comments next. FYI if you press "/" 3 times in visual studio it creates a summary comment section that then gets added to the description of the method/event/property when intellisense pops up.

Reply:
Okay, no problem.

Reply:
Will be solved in #8